Source/EbsdLib/TSL/AngReader.cpp

```diff
@@ -568,56 +568,67 @@ void AngReader::parseHeaderLine(QByteArray& buf)
   // parsing data for the phase then stick the Phase instance into the header
   // map or stick it into a vector<Phase::Pointer> and stick the vector into
   // the map under the "Phase" key
-  if(word.compare(Ebsd::Ang::Phase) == 0)
+  if(word == Ebsd::Ang::Phase)
   {
     m_CurrentPhase = AngPhase::New();
-    m_CurrentPhase->setPhaseIndex(tokens.at(1).toInt(&ok, 10));
+    int32_t phaseIndex = tokens.at(1).toInt(&ok, 10);
```

Comment:
There's no error checking here for tokens unlike the other cases. Also does the comment version of this line always have a non integer token?